### DISCUSSION TITLE
Custom SSL certificate

### DISCUSSION THREAD

**Discussion by Shaquu** [community/community#22443] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,<br>
I need to use my own ssl certificate on page hosted via GitHub Pages.<br>
Is it possible? If yes then how can I install it?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/shaquu">@Shaquu</a>, welcome to the GitHub Support Community!</p>
<p>It’s not possible to use a custom SSL certificate with a Pages site I’m afraid. Pages only supports SSL certificates that it generates for you automatically.</p>

**Comment by Shaquu** [comment]
<p><a class="mention" href="/thomasshaped">@thomasshaped</a> I am not satisfied but thank you for confirmation.</p>

**Comment by hyekangjoo** [comment]
<p>Would you mind helping me with this please?: <a href="https://github.community/t/how-to-point-github-io-website-to-a-custom-domain-with-ssl-installed/207292" class="inline-onebox">How to point github.io website to a custom domain with SSL installed?</a></p>
